micrometer-core/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
 
     compile 'org.mongodb:mongo-java-driver:latest.release', optional
 
+    compile 'org.jooq:jooq-meta:latest.release', optional
```

Comment:
I think you wouldn't need `jooq-meta` as a dependency. It's the auxiliary library that provides schema meta information to the code generator. It has a transitive dependency on `org.jooq:jooq`, which is the one you probably want to depend on, directly.

Reply:
Changed in f2814031353d0f96e1daf1a3ca204e89f540f6c2. Thanks